### PR TITLE
refactor: Move from type comments to inline type annotations.

### PR DESCRIPTION
**Pull request by ahmedabuamra**
Ran `com2ann` in the project folder to auto re-write the files.

Fixes: #901

### CODE REVIEW

File: zulipterminal/model.py [hunk 1]
```diff
@@ -458,10 +460,13 @@ def update_stream_message(self, topic: str, message_id: int,
...
-                                         'type': 'realm_emoji'}
-                         for emoji_code, emoji in response['emoji'].items()
-                         if not emoji['deactivated']}  # type: NamedEmojiData
+        custom_emojis: NamedEmojiData = {
+            emoji['name']: {
+                'code': emoji_code,
+                'type': 'realm_emoji'
+                }
+            for emoji_code, emoji in response['emoji'].items()
+            if not emoji['deactivated']}
```
Comment (neiljp) [inline zulipterminal/model.py:469]: Did com2ann generate this new layout? We're moving towards a style where large bracketed code have the closing one on the starting line, though the original version isn't ideal in this respect either.

File: zulipterminal/ui.py [hunk 2]
```diff
@@ -94,8 +94,8 @@ def set_typeahead_footer(self, suggestions: List[str],
                              is_truncated: bool) -> None:
         if suggestions:
             # Wrap by space.
-            footer_text = [' ' + s + ' '
-                           for s in suggestions]  # type: List[Any]
+            footer_text: List[Any] = [' ' + s + ' '
+                                      for s in suggestions]
```
Comment (neiljp) [inline zulipterminal/ui.py:98]: Fits on one line?

**Review by neiljp — COMMENTED** [review]
@ahmedabuamra Welcome to the project and thanks for looking into this :+1:

The com2ann tool is automated so I've not looked in fine detail, but noticed a few cases we may want to look into in terms of formatting, checking the state of lister/run-mypy compared to zulip/zulip, and OrderedDict (as discussed on chat.zulip.org).

This does point to various cases where we could improve the typing - while equivalent I'd prefer `Optional[X]` to `Union[None, X]`, for example - but we can address those in another PR.

File: zulipterminal/model.py (hunk 1 above)
Comment (ahmedabuamra) [reply zulipterminal/model.py:469]: Meaning to be like that, right?

```
        custom_emojis: NamedEmojiData = {
            emoji['name']: {
                'code': emoji_code,
                'type': 'realm_emoji'
            }
            for emoji_code, emoji in response['emoji'].items()
            if not emoji['deactivated']
        }
```

File: tools/lister.py [hunk 3]
```diff
@@ -69,8 +69,8 @@ def list_files(targets=[], ftypes=[], use_shebang=True, modified_only=False,
     # throw away empty lines and non-files (like symlinks)
     files = list(filter(os.path.isfile, files_gen))
 
-    result_dict = defaultdict(list)  # type: Dict[str, List[str]]
-    result_list = []  # type: List[str]
+    result_dict: Dict[str, List[str]] = defaultdict(list)
+    result_list: List[str] = []
```
[the remark this answers is not in the document]
Comment (ahmedabuamra) [reply tools/lister.py:73]: I understand. `tools/lister.py` is the only changed file in `tools`.

**Comment by ahmedabuamra** [comment]
@zulipbot remote "PR needs update"

**Comment by ahmedabuamra** [comment]
@zulipbot remove "PR needs update"

**Comment by andersk** [comment]
To deal with types like `OrderedDict[K, V]` that parse in mypy but don’t evaluate in Python at runtime, you can quote them:

```python
KEY_BINDINGS: "OrderedDict[str, KeyBinding]" = OrderedDict([
    …
])
```

File: zulipterminal/model.py (hunk 1 above)
Comment (neiljp) [reply zulipterminal/model.py]: Right :+1: Also trailing commas, but that's simpler to quickly fix :)

**Comment by ahmedabuamra** [comment]
Thanks @andersk, I updated the PR as you suggested and waiting for @neiljp 's confirmation too.

File: zulipterminal/model.py (hunk 1 above)
Comment (ahmedabuamra) [reply zulipterminal/model.py]: @neiljp Done :)

**Comment by neiljp** [comment]
@ahmedabuamra Thanks for getting this done - I'm merging this now :tada:

**Pull request merged**